subsys/bluetooth/audio/bap_broadcast_source.c

```diff
@@ -726,12 +780,32 @@ int bt_bap_broadcast_source_create(struct bt_bap_broadcast_source_param *param,
 		for (size_t j = 0U; j < subgroup_param->params_count; j++) {
 			const struct bt_bap_broadcast_source_stream_param *stream_param;
 			struct bt_bap_stream *stream;
+			struct bt_audio_codec_cfg *codec_cfg;
 
+			codec_cfg = subgroup_param->codec_cfg;
 			stream_param = &subgroup_param->params[j];
 			stream = stream_param->stream;
 
+			if (CONFIG_BT_AUDIO_CODEC_CFG_MAX_DATA_SIZE > 0) {
+				if (bis_count >= BROADCAST_STREAM_CNT) {
+					LOG_DBG("Stream count %d exceeded", bis_count);
+					return -ENOMEM;
+				}
+				codec_cfg = &source->codec_cfg[bis_count];
+				memcpy(codec_cfg, subgroup_param->codec_cfg,
+				       sizeof(struct bt_audio_codec_cfg));
+				err = update_codec_cfg_data(codec_cfg, stream_param);
+				if (err != 0) {
```

Comment:
```suggestion
				}
				
				codec_cfg = &source->codec_cfg[bis_count];
				memcpy(codec_cfg, subgroup_param->codec_cfg,
				       sizeof(struct bt_audio_codec_cfg));
				       
				err = update_codec_cfg_data(codec_cfg, stream_param);
				if (err != 0) {
```
Suggest to add some "air" around code blocks